go/obscurocommon/l1_types.go

```diff
@@ -2,12 +2,71 @@ package obscurocommon
 
 import (
 	"math/big"
+	"math/rand"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
 	"github.com/ethereum/go-ethereum/trie"
 )
 
+// L1TxType - Just two types of relevant L1 transactions: Deposits and Rollups
```

Comment:
does this bit here need to be back?

Reply:
Good spot - yeah that got lost in the swamp